src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_DeconstructionAssignmentOperator.cs

```diff
@@ -129,18 +129,12 @@ private void ApplyAssignments(BoundDeconstructionAssignmentOperator node, ArrayB
         }
 
         /// <summary>
-        /// Makes an expression that constructs the return value for the deconstruction.
-        /// For d-declarations, that is simply void.
-        /// For d-assignments, that is a series of tuple constructions, that are chained with the help of placeholders.
+        /// Makes an expression that constructs the return value for the deconstruction:
+        /// a series of tuple constructions, that are chained with the help of placeholders.
         /// The placeholders that are set are added to the list for later clearing.
         /// </summary>
         private BoundExpression MakeReturnValue(BoundDeconstructionAssignmentOperator node, ArrayBuilder<BoundValuePlaceholderBase> placeholders)
```

Comment:
Do we check whether the returned value is used so we can avoid creating it if not used?